docs/blueprints/EP031.rst

```diff
@@ -39,7 +39,7 @@ This blueprint has the following characteristics:
   4. There will be no concerns about proxy ports and their mappings. These were addressed in Blueprint EP034.
   5. There is no need for persistent data. The **mef_eline** and **flow_manager** napps will persist their entries accordingly since **telemetry_int** will leverage **flow_manager**.
   6. This version won't require changes to the way the **mef_eline** napp works. However, a new value will be added each EVC's metadata attribute.
-  7. This specification assumes the data plane's pipeline is ready for INT, with multiple tables, and it assumes that **mef_eline** uses table 0. **telemetry_int** aims to use any table with an ID higher than **mef_eline**, for instance in this document, table 2.
+  7. This specification assumes the data plane's pipeline is ready for INT, with multiple tables, and it assumes that **mef_eline** uses table 0. **telemetry_int** aims to use any table with an ID higher than **mef_eline**, for instance in this document, table 2. It also assumes that this higher table should be capable of exact matching on ``in_port`` and ``dl_vlan``, including on just one of these fields, just so both EVPLs and EVPLs flows are correctly supported.
```

Comment:
cc'ing @italovalcy @jab1982 regarding the requirement to support also EPL on table `"n"`. I left the requirement still higher level, but looks like we'll probably go for another table just for EPLs. I'll leave it up to you guys to confirm, since this is highly dependent on pipeline configuration architecture. The implementation to support another table shouldn't be too difficult, although if we still had a single one it'd be even easier to maintain and test.

If we do end up introducing a new table, we should also think about the concept of table group, that we introduced on [`of_multi_table` NApp](https://github.com/kytos-ng/of_multi_table/blob/master/openapi.yml#L164-L169), which in summary is reponsible for categorizing a set of flows that should always be installed in a particular table. So far `telemetry_int` only had a `base` group table (since up to this point it only required one extra table), so with a new table we probably want to split this into `int_evpl` and `int_epl` since it'll still have to follow `mef_eline`'s table groups `evpl` and `epl` which currently default to table 0 anyway.

Reply:
- EVPLs will use table 2 by default
- EPLs will use table 3 by default

Updated the text on https://github.com/kytos-ng/kytos/pull/402/commits/2096fc5da5e675562feba9df0b584acd2e4818c5